DigitalLearningSolutions.Data/DataServices/UserDataService.cs

```diff
@@ -82,7 +82,7 @@ public UserDataService(IDbConnection connection)
                         au.SummaryReports,
                         au.UserAdmin AS IsUserAdmin,
                         au.CategoryID,
-                        cc.CategoryName,
+                        ISNULL(cc.CategoryName, 'All') AS CategoryName,

```

Comment:
I presume this is because if an AdminUser has the rights for all categories, they have a categoryId of 0, and thus the CategoryName will be NULL as no category will match the Id of 0. 

I'm wondering if it might be better do some something a little more specific, like a CASE that returns the CategoryName if au.CategoryId != 0, and "All" if it is 0. My worry is that while we know CategoryId = 0 definitely returns a NULL CategoryName, it might also be returned in other circumstances, like if some data gets messed up and a non-zero but non-existent categoryid is used. But I'm not sure if its worth it, as I don't think that's a particularly likely scenario.